Beta_Integrations/Telegram/CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 ## [Unreleased]
+-
 Updated the integration's image.
```

Comment:
How about this? 
```
## [Unreleased]
Updated the integration's image.
```

Reply:
lets just delete that line, and keep `-`